spec/requests/admin/repositories.rb

```diff
@@ -0,0 +1,93 @@
+require 'rails_helper'
+
+describe 'Admin::Repositories', type: :request do
```

Comment:
I think this should be a model test (unit) instead of a request test (integration). We just want to test that when calling `repository.destroy` the events aren't destroyed, there is no need for a request or anything else

Reply:
I'm doing the unit test. You think is worth it to leave this test since repositories will be mainly deleted from the admin or isn't necessary?

Reply:
I think the changes on this PR are only related to the model, there isn't anything special in the admin or controller